src/Microsoft.ML.Transforms/LambdaTransform.cs

```diff
@@ -69,6 +69,7 @@ private static ITransformer Create(IHostEnvironment env, ModelLoadContext ctx)
             var contractName = ctx.LoadString();
 
             var composition = env.GetCompositionContainer();
+            Contracts.CheckValue(composition,nameof(composition));
```

Comment:
Note that this is being deleted with #2569

Reply:
I understand. 
I'm just trying to resolve this comment from TestCustomTrasformer :
`we should have a common mechanism that will make sure this is 'our' exception thrown.`

Only way I know how to do that is to check was exception marked or not.
Currently composition is null, and in next line it throws NullReference. Which is unmarked, and in order to make it marked, I need to run check on it.